docker/centos-7/Dockerfile

```diff
@@ -151,7 +152,7 @@ RUN cp /usr/local/lib/libgtest.a /lib64/
 
 # install specific arakoon.
 RUN git clone https://github.com/openvstorage/arakoon.git
-RUN cd arakoon && git pull && git checkout 1.9.21
+RUN cd arakoon && git pull && git checkout d4f257272b990c36949bc305485c30a672180980
```

Comment:
it's cleaner to checkout tags/1.9.22 instead? (here and in the other dockerfiles)